pulse2percept/electrode2currentmap.py

```diff
@@ -166,24 +166,32 @@ def retinalmovie2electrodtimeseries(rf, movie, fps=30):
     return rflum
 
 def get_pulse(pulse_dur, tsample, interphase_dur, pulsetype):
+    """Returns a single biphasic pulse.
+
+    A single biphasic pulse with duration `pulse_dur` per phase, separated
+    by `interphase_dur` is returned.
+    Pulse type is either `cathodicfirst` (negative phase first) or
+    `anodicfirst` (positive phase first).
```

Comment:
While you're at it (thanks for adding the docstring!), would you mind using the numpy docstring standard? https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt